app/src/forms/file/service.js

```diff
@@ -1,39 +1,69 @@
 const config = require('config');
 const { v4: uuidv4 } = require('uuid');
 
+const Queue = require('bull');
+
 const { FileStorage } = require('../common/models');
 const storageService = require('./storage/storageService');
 
 const PERMANENT_STORAGE = config.get('files.permanent');
 
+// create our job queue
+const queue = new Queue('Upload files queue', { redis: { password: 'redispass' } });
+
 const service = {
   create: async (data, currentUser) => {
-    let trx;
-    try {
-      trx = await FileStorage.startTransaction();
+    const job = await queue.add('Upload files job', {
+      fileData: data,
+      user: currentUser,
+    });
 
-      const obj = {};
-      obj.id = uuidv4();
-      obj.storage = 'uploads';
-      obj.originalName = data.originalname;
-      obj.mimeType = data.mimetype;
-      obj.size = data.size;
-      obj.path = data.path;
-      obj.createdBy = currentUser.usernameIdp;
+    queue
+      .on('completed', function () {
+        console.log('Upload file Job completed...');
```

Comment:
Most of these console.log statements need to be removed

Reply:
agree for success completion but still I think need leave if it fails for logging

Reply:
We should look into using this log.
https://github.com/bcgov/common-hosted-form-service/blob/master/app/src/components/log.js